### ISSUE TITLE
[lib.cuckoo.core.plugins] WARNING: The reporting module "Malheur" returned the following error: Failed to perform Malheur classification: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by neex01** [spender-sandbox/cuckoo-modified#30]
I am getting this error for all samples.  Nonetheless, for each analysis, a Malheur report gets saved to storage\malheur\reports\.

**Comment by spender-sandbox** [comment]
Are you running the latest version of Malheur from https://github.com/rieck/malheur ? We changed to using their new config format, so it will fail if you're using an older version.

-Brad

**Comment by neex01** [comment]
Yes, version 0.5.4. However, I am running from OS X--not sure if this might be the issue..?

**Comment by spender-sandbox** [comment]
Possibly -- I can't test on there.  Can you run the malheur command present in the reporting module manually and see what error it's giving?

-Brad

**Comment by neex01** [comment]
Sure.. it doesn't seem to like the -c command-line option to reference the config file..

**Comment by neex01** [comment]
resolved: edited malheur.py with following changes..

line 148: cfgpath = os.path.join(CUCKOO_ROOT, "cons")
line 167: cmdline = ["malheur", "-m", cfgpath, "-o", outputfile, "cluster", reportsdir]

**Comment by spender-sandbox** [comment]
You're not running the latest version (via git HEAD, not version number -- the changes I mentioned above were after the bump to 0.5.4)?  There is no -m option in the latest version, and the -c option was added via this commit: https://github.com/rieck/malheur/commit/28e03686da7360bb5717aacf1d55327f07c7004d

-Brad

**Comment by neex01** [comment]
right you are!  working now with latest and greatest version of Malheur.  many thanks.

**Comment by neex01** [comment]
After analyzing 178 samples, I'm getting this same error again.  While restarting Cuckoo does not help, a total clean of jobs/samples/data/etc. does resolve the issue... for a while, then it consistently prints this error after 178 samples.

**Comment by spender-sandbox** [comment]
That could be a different issue (there are a couple bugs in Malheur that I had fixed upstream, this is also why we require the current git HEAD).  Can you re-run the commandline and see what the problem is?

-Brad

**Comment by neex01** [comment]
Ran the command-line, and Malheur completed with no problems against the (>178) reports I generated with Cuckoo.

Note that, even when it prints the error after 178 samples, the individual report files are being generated in storage/malheur/reports, but everything after sample 178 is not being saved to storage/malheur/malheur.txt.

As a next step, I am going to run it against a different batch of samples..

**Comment by neex01** [comment]
The other batch got well past 178.  I then threw sample 178 from the original batch into another batch and, as bizarre as it sounds, that same sample is what killed the Malheur reporting (this time as sample 33).

**Comment by garanews** [comment]
I'm having same issue too, after some time when malheur was working correctly, now I'm receiving that error:
WARNING: The reporting module "Malheur" returned the following error: Failed to perform Malheur classification: [Errno 2] No such file or directory
I'm using latest version etc...

**Comment by garanews** [comment]
But for other analysis, exmaple  the file 11323.txt under /storage/malheur/reports is generated..

**Comment by dperrie20** [comment]
Hi, I am having the same issue getting the error: 
WARNING: The reporting module "Malheur" returned the following error: Failed to perform Malheur classification: [Errno 2] No such file or directory

Did you guys come to a solution as I believe I am using the latest version of malheur.
